orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/ModifyScalingGroupStage.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Alibaba Group.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.kato.pipeline
+
+
+import com.netflix.spinnaker.orca.kato.tasks.ModifyScalingGroupTask
+import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
+import com.netflix.spinnaker.orca.pipeline.TaskNode
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import groovy.transform.CompileStatic
+import org.springframework.stereotype.Component
+
+@Component
+@CompileStatic
+class ModifyScalingGroupStage implements StageDefinitionBuilder {
+
+  @Override
+  void taskGraph(Stage stage, TaskNode.Builder builder) {
+    builder.withTask("modifyScalingGroup", ModifyScalingGroupTask)
+  }
+
+}
```

Comment:
Please do not add new Groovy files to the `main` source set.

Reply:
I am sorry. I can not get it. Do you mean I need to change `Groovy Project` to `Java Project`?

Reply:
HI . @robzienert Can you give me some ideas?

Reply:
Outside of `test` classes, submitted classes must not be Groovy: Use Java instead.